ppa/archive/management/commands/index.py

```diff
@@ -70,8 +69,14 @@ def add_arguments(self, parser):
             'source_ids', nargs='*',
             help='Optional list of specific works to index by source (HathiTrust) id.')
         parser.add_argument(
-            '-i', '--index', choices=['all', 'works', 'pages'],  default='all',
+            '-i', '--index', choices=['all', 'works', 'pages'], default='all',
             help='Index only works or pages (by default indexes all)')
+        parser.add_argument(
```

Comment:
r&d day thought: allow passing any qualified model name as an arg, e.g.
```py 
python manage.py index ppa.archive.works
```
throw an error if the model doesn't implement `Indexable`.